lisa/tools/firewall.py

```diff
@@ -16,19 +16,19 @@ def can_install(self) -> bool:
         return False
 
     def stop(self) -> None:
-        cmd_result = self.node.execute("command -v ufw")
+        cmd_result = self.node.execute("command -v ufw", shell=True)
```

Comment:
What's the reason it needs the shell?

Reply:
This step was failing on ubuntu. It was unable to find `ufw` but after adding shell it works fine